### DISCUSSION TITLE
Can a block contain multiple block handlers

### DISCUSSION THREAD

**Discussion by Tofaa2** [Minestom/Minestom#1728] [Help] [answered]
Basically what the title says

**Comment by KrystilizeNevaDies** [comment, marked as answer]
No it cannot.
A solution would be to make your block handlers have a fallback to the previous block handler (if any).

**Reply by Tofaa2** [reply]
Wdym? I was just asking so i dont have to define it for every single block, but i suppose its okay

**Reply by KrystilizeNevaDies** [reply]
Oh, I would recommend just having a static block instance with your handler in that case.
